src/storage/hummock_test/src/bin/replay/replay_impl.rs

```diff
@@ -206,10 +206,7 @@ impl LocalReplay for LocalReplayImpl {
     }
 
     fn seal_current_epoch(&mut self, next_epoch: u64, opts: TracedSealCurrentEpochOptions) {
-        self.0.seal_current_epoch(
-            next_epoch,
-            opts.try_into().expect("should not fail to convert"),
-        );
+        self.0.seal_current_epoch(next_epoch, opts.into());
```

Comment:
Why we have this? Is `TryFrom` derived from `From`? 🤔

Reply:
Yes

```
// Infallible conversions are semantically equivalent to fallible conversions
// with an uninhabited error type.
#[stable(feature = "try_from", since = "1.34.0")]
impl<T, U> TryFrom<U> for T
where
    U: Into<T>,
{
    type Error = Infallible;

    #[inline]
    fn try_from(value: U) -> Result<Self, Self::Error> {
        Ok(U::into(value))
    }
}
```